pkg/scheduler/internal/podinfo/clock.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package podinfo
```

Comment:
I forgot we had clock stuff in here. I should probably file a separate issue to delete that. :smile:

Reply:
@misterikkit We should delete the clock stuff or should we migrate it into another package ?

Reply:
For this PR, I'd like to keep the clock stuff in the podinfo package.

After this PR, I want to delete it since there is already clock functionality in https://godoc.org/k8s.io/apimachinery/pkg/util/clock.